### PR TITLE
friend remover v0.1

### PR DESCRIPTION
**Pull request by toxicwave1**

### CODE REVIEW

File: store/extensions/Friend Remover/extension.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    "source": "https://github.com/Tragic7432/Friend-Remover",
+    
+    "stable": true,
+    
+    "framework": {
+      "name": "G-Python",
+      "version": "0.1.5"
+    },
+
+    "language": "Java",
```
Comment (Alynva) [inline store/extensions/Friend Remover/extension.json]: language should be "Python"

**Review by Alynva — CHANGES_REQUESTED** [review]

**Comment by Alynva** [comment]
i don't know a lot of python, but the handler souldn't be like

```python
def start(self, s):
```
?

File: store/extensions/Friend Remover/extension.json (hunk 1 above)
Comment (toxicwave1) [reply store/extensions/Friend Remover/extension.json]: done

**Review by Alynva — APPROVED** [review]

**Comment by toxicwave1** [comment]
> i don't know a lot of python, but the handler souldn't be like
> 
> ```python
> def start(self, s):
> ```
> 
> ?

I dont use init method so no need for "self".

**Comment by sirjonasxx** [comment]
`friendremover.zip` -> `extension.zip`

**Comment by Alynva** [comment]
> `friendremover.zip` -> `extension.zip`

That file should be removed. Another `extension.zip` already exists

**Pull request merged**